router.go

```diff
@@ -112,14 +112,20 @@ func setupRoutes(router *mux.Router, oas *OpenAPISpec, env config.EnvironmentVar
 	// NOTE: The following sort is required by mux router because it expects
 	// routes to be registered in the proper order
 	paths := make([]string, 0)
-	methods := make(map[string][]string, len(oas.Paths))
+	methods := make(map[string][]string, 0)
+
 	for path, pathMethods := range oas.Paths {
 		paths = append(paths, path)
 		for method := range pathMethods {
+			if method == AllHTTPMethod {
+				methods[path] = OasSupportedHTTPMethods
```

Comment:
are you sure about this?
what if someone defines both "all" and "get" method?

Reply:
I don't see a problem, if you define `all` and `get` is because you need different policies, this discrimination is done by bunrouter in a different part of the code, here we're just recording all routes and methods we have to expose as APIs and when you say you want `all` it means we have to register all methods regardless of any other specific methods.

We could discuss that in this circumstance we would register `get` twice, but I don't think it would be a problem, is it? @Evange95

Reply:
inside the OAS file there is a case with all and get method. 
In that case what happens? skip other method because it is present "all"?

Reply:
I will not be worried about it, the method will be correctly registered. My worry's about a possible double method registration inside the methods[path] if we have bot an `all` and a `get` for example. Anyway, defining which policy to evaluate is not the job of the mux router. I'm going to add a small guard for this situation

Reply:
A set structure can be used to avoid the double registration.

Reply:
does the registration actually gets doubled or does mux takes this into account?

Reply:
I verified the mux code and it implicitly ignores duplicates so I don't think it's a huge problem